src/ralph/cli.py

```diff
@@ -437,24 +433,52 @@ def list_(details, new, backend, **options):
         logger.warning("Configured %s backend contains no archive", backend)
 
 
-@cli.command()
-def runserver():
-    """
-    Run the API server for the development environment. Starting uvicorn
-    programmatically for convenience and documentation.
+@backends_options(name="runserver", backends=API_DATABASE_BACKENDS)
+@click.option(
+    "-h",
+    "--host",
+    type=str,
+    required=False,
+    default=RUNSERVER_HOST,
+    help="Bind socket to this host",
+)
+@click.option(
+    "-p",
+    "--port",
+    type=int,
+    required=False,
+    default=RUNSERVER_PORT,
+    help="Bind socket to this port",
+)
+def runserver(host, port, backend, **options):
+    """Runs the API server for the development environment.
+
+    Starts uvicorn programmatically for convenience and documentation.
     """
-    logger.info("Running API server on %s:%s.", RUNSERVER_HOST, RUNSERVER_PORT)
+
+    logger.info("Running API server on %s:%s with %s backend", host, port, backend)
     logger.info(
         (
             "Do not use runserver in production - start production servers "
             "through a process manager such as gunicorn/supervisor/circus."
         )
     )
-    uvicorn.run(
-        "ralph.api:app",
-        host=RUNSERVER_HOST,
-        port=RUNSERVER_PORT,
-        log_level="debug",
-        reload=True,
-    )
+
+    with NamedTemporaryFile(mode="w+", encoding="utf-8") as temp_env_file:
```

Comment:
Why opening this file in `w+` mode?

Reply:
Well spotted! Thanks) Indeed, just `w` mode is enough) I thought `w+` would be needed for reading in this case, but it's not true)